### PR TITLE
Ryna - L111.nonghg_en_R_S_T_Y

### PR DESCRIPTION
**Pull request by rynacui**

### CODE REVIEW

File: R/zchunk_L111.nonghg_en_R_S_T_Y.R [hunk 1]
```diff
@@ -60,8 +69,153 @@ module_emissions_L111.nonghg_en_R_S_T_Y_DISABLED <- function(command, ...) {
...
+    L111.nonghg_tgej_USA_en_Sepa_F_Yh <- bind_rows(L101.co_tgej_USA_en_Sepa_F_Yh %>% mutate(Non.CO2 = "CO"),
+                                                   L101.so2_tgej_USA_en_Sepa_F_Yh %>% mutate(Non.CO2 = "SO2"),
+                                                   L101.nox_tgej_USA_en_Sepa_F_Yh %>% mutate(Non.CO2 = "NOx"),
+                                                   L101.voc_tgej_USA_en_Sepa_F_Yh %>% mutate(Non.CO2 = "NMVOC"),
+                                                   L101.nh3_tgej_USA_en_Sepa_F_Yh %>% mutate(Non.CO2 = "NH3")) %>%
+      arrange(Non.CO2, sector, fuel, year) %>%
+      # For each gas, sector and fuel, replace 2003-2008 data with 2002 values
+      group_by(Non.CO2, sector, fuel) %>%
+      mutate(value = replace(value, year >= 2003 & year <= 2008, value[year == 2002])) %>%
+      rename(emfact = value)
```
Comment (bpbond) [inline R/zchunk_L111.nonghg_en_R_S_T_Y.R]: Missing `%>%` at the end of this line?

**Comment by rynacui** [comment]
@kdorheim please add the old system behavior to address issue #933

File: R/zchunk_L111.nonghg_en_R_S_T_Y.R [hunk 2]
```diff
@@ -60,8 +69,153 @@ module_emissions_L111.nonghg_en_R_S_T_Y_DISABLED <- function(command, ...) {
...
+    # First, calculate total energy use in international shipping and aviation
+    L111.nonghg_tg_R_en_Si_F_Yh <- L111.nonghg_tg_R_en_Si_F_Yh_intl %>%
+      group_by(EDGAR_agg_sector, Non.CO2, year) %>%
+      summarise(tot_energy = sum(energy)) %>%
+      ungroup %>%
+      # Combine total energy with energy use by region
+      right_join(L111.nonghg_tg_R_en_Si_F_Yh_intl, by = c("EDGAR_agg_sector", "Non.CO2", "year")) %>%
+      # Match in EDGAR international shipping and international aviation emissions
+      left_join(L111.EDGAR_intl, by = c("EDGAR_agg_sector", "Non.CO2", "year")) %>%
+      mutate(tot_emiss = tot_emiss / 1000.0,
```
Comment (bpbond) [inline R/zchunk_L111.nonghg_en_R_S_T_Y.R]: Maybe a comment here to clarify? What's the unit change happening here?

**Comment by kdorheim** [comment]
@rynacui thank you so much! I'll add an old new behavior block around here

**Comment by bpbond** [comment]
All tests now pass. 🎉

**Pull request merged**